PWGCF/FemtoDream/FemtoDreamV0Selection.h

```diff
@@ -130,7 +130,8 @@ class FemtoDreamV0Selection : public FemtoDreamObjectSelection<float, femtoDream
   static int findSelectionIndex(const std::string_view& obs, std::string_view prefix = "")
   {
     for (int index = 0; index < kNv0Selection; index++) {
-      std::string_view cmp{static_cast<std::string>(prefix) + static_cast<std::string>(mSelectionNames[index])};
+      std::string comp = static_cast<std::string>(prefix) + static_cast<std::string>(mSelectionNames[index]);
+      std::string_view cmp{comp};
```

Comment:
Any reason for not keeping the previous version?

Reply:
The reason is the same reported in this [PR](https://github.com/AliceO2Group/O2Physics/commit/b4c6602aec055dbada65bee8d4ad4577aa98bda4). Basically, it does not work properly if done separately

Reply:
Just one and a half months and already forgot about it!!!